hack/env.sh

```diff
@@ -1,22 +1,33 @@
-if ! skopeo -v &> /dev/null
-then
-        echo "skopeo could not be found"
-        exit 1
+if [ -z $SKIP_VAR_SET ]; then
```

Comment:
is the name fine or should it be named SRIOV_SKIP_VAR_SET ?

Reply:
It's fine.